odl/set/domain.py

```diff
@@ -218,23 +246,44 @@ def approx_contains(self, point, tol):
         True
         """
         point = np.atleast_1d(point)
-        if point.dtype == object:
-            return False
-        if np.any(np.isnan(point)):
-            return False
-        if point.ndim > 1:
-            return False
-        if len(point) != self.ndim:
-            return False
-        if point[0] not in RealNumbers():
+        if point.shape != (self.ndim,):
             return False
-        if self.dist(point, ord=np.inf) > tol:
+        if not is_real_dtype(point.dtype):
             return False
-        return True
+        return self.dist(point, ord=np.inf) <= tol
 
     def __contains__(self, other):
-        """Return ``other in self``."""
-        return self.approx_contains(other, tol=0)
+        """Return ``other in self``.
+
+        Parameters
+        ----------
+        other
```

Comment:
Nope, but this should work I think, will do a big test of doc before we merge conebeam